src/v2/Components/ArtworkFilter/ArtworkFilters/SizeFilter.tsx

```diff
@@ -66,7 +65,7 @@ export const parseRange = (range: string = "", metric: Metric): Numeric[] => {
     if (s === "*") return s
     const value = parseFloat(s)
 
-    if (metric === "cm") {
+    if (metric === "CM") {
```

Comment:
This makes me think - is there a reason we're doing these values as `CM` instead of `cm`? I think it's fine either way, we should just make sure that Gravity can handle both uppercase and lowercase 🤔

Reply:
> is there a reason we're doing these values as CM instead of cm

I decided to store values in Force in the same format as they are stored on Gravity side